packages/ui/src/app/pages/Proposals/CurrentProposals.stories.tsx

```diff
@@ -781,6 +781,46 @@ export const SpecificParametersFundingRequest: Story = {
   }),
 }
 
+export const SpecificParametersMultipleFundingRequest: Story = {
+  play: specificParametersTest('Funding Request', async ({ args, createProposal, modal, step }) => {
+    const bob = member('bob')
+    await createProposal(async () => {
+      const nextButton = getButtonByText(modal, 'Create proposal')
+      expect(nextButton).toBeDisabled()
+
+      await userEvent.click(modal.getByTestId('pay-multiple'))
+
+      const csvField = modal.getByTestId('accounts-amounts')
+
+      // Invalid
+      await userEvent.clear(csvField)
+      await userEvent.type(csvField, `${alice.controllerAccount},500${bob.controllerAccount},500`)
+      expect(modal.findByText(/Not valid CSV format/))
+      expect(nextButton).toBeDisabled()
+
+      // Valid
+      await userEvent.clear(csvField)
+      await userEvent.type(csvField, `${alice.controllerAccount},500\n${bob.controllerAccount},500`)
```

Comment:
Could you add an invalid accounts, a line where the amount is too high, and some more lines (and decrease the value of `fundingRequestProposalMaxAccounts`, in the mocks) too get the "Maximum allowed accounts exceeded" error please ? Then the test can check the errors show correctly, then remove the account from the `PreviewAndValidateModal`. But before that: the test should also close the `PreviewAndValidateModal` and make sure the "Create proposal" button is still disabled.